assets/js/hooks/metamask.js

```diff
@@ -0,0 +1,56 @@
+import {ethers} from "../../vendor/ethers"
+
+const web3Provider = new ethers.providers.Web3Provider(window.ethereum)
+
+function init() {
+    // Check that the web page is run in a secure context,
+    // as otherwise MetaMask won't be available
+    if (location.hostname !== "localhost" && location.protocol !== 'https:') {
+        console.log("FAILING AT HTTPS")
+    }
+}
```

Comment:
This is not really checking that the page is in a secure context, it's just printing out stuff to the console. Are you planning to refactor this later on?

Reply:
No, this was actually a piece of code from the original repo where I got the base. I can remove it, though...